sites/website/src/docs/design-systems/high-contrast.md

```diff
@@ -0,0 +1,194 @@
+---
+id: high-contrast
+title: High Contrast in FAST
+sidebar_label: High Contrast
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/high-contrast.md
+---
+
+## Styling components using forced-colors.
+High contrast mode uses the CSS media feature, `forced-colors`. When `forced-colors` is set to active, the user agent will apply a limited color palette to the component.
+
+**Example:**
+```css
+@media (forced-colors: active) {
+    :host {
+        background: ButtonFace;
+    }
+}
+```
+
+FAST has a utility function that is use to construct `forced-colors` in the stylesheet, called [forcedColorsStylesheetBehavior()](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/utilities/match-media-stylesheet-behavior.ts). This is added in the `withBehavior()` function inside the `css` style.
```

Comment:
Lets add a note here about *why* this behavior exists - something along the lines of:

"To avoid the runtime cost applying forced-color style rules when the UA does not match the forced-colors @media query, FAST exposes a behavior that conditionally adds and removes stylesheets based on the forced-colors @media query. forced-colors stylesheets can then be conditionally applied where necessary."